src/main/java/games/strategy/engine/config/client/remote/RemoteFileFetcher.java

```diff
@@ -0,0 +1,70 @@
+package games.strategy.engine.config.client.remote;
+
+import java.io.IOException;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import games.strategy.engine.framework.map.download.DownloadUtils;
+import games.strategy.engine.lobby.client.login.LobbyServerProperties;
+import games.strategy.util.Version;
+
+public class RemoteFileFetcher {
```

Comment:
I was a bit confused when I first saw this class.  I thought it was some kind of general remote file fetcher, but then I realized it was specific to the lobby properties file.  Since it's not in a lobby-specific package, would it be appropriate to include "lobby" somewhere in its name (e.g. `RemoteLobbyPropertyFileFetcher`)?  Or am I missing the intent?